attic/rfc_metrics.md

```diff
@@ -36,14 +51,55 @@ Internally, the function would look the following:
 
 Example:
 ```r
-fn = function(truth, response, positive, ...) {
-  assert_binary(truth, response = response, positive = positive)
-  fn_cm(cm(truth, response, positive))
-}
+MeasureFairness = R6Class("MeasureFairness", inherit = Measure, cloneable = FALSE,
+  public = list(
+    fun = NULL,
+    na_value = NaN,
+    base_measure = NULL,
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function(name, base_measure) {
+      info = mlr3fairness::measures[["groupwise_abs_diff"]]
+      super$initialize(
+        id = paste0("fairness.", base_measure$id),
+        range = c(info$lower, info$upper),
+        minimize = info$minimize,
+        predict_type = info$predict_type,
+        packages = "mlr3fairness",
+        man = paste0("mlr_measures_fairness.", name)
+      )
+      self$fun = get(name, envir = asNamespace("mlr3fairness"), mode = "function")
```

Comment:
In the implementation, the function can live inside the class I think

Reply:
I am not sure whether I understand it here. Which class do you mean here?

Reply:
Can we not define a member function `fun` instead of setting it here?